tb_plugin/torch_tb_profiler/profiler/module_op.py

```diff
@@ -0,0 +1,220 @@
+from copy import copy
+
+from .node import ModuleNode, ProfilerStepNode
+from .trace import EventTypes
+
+
+class Module:
+    def __init__(self, name, module_id):
+        self.name = name
+        self.module_id = module_id
+        self.children = []
+
+    def __hash__(self):
+        if self.children:
```

Comment:
nit: not too familiar with Python but what happens if you just `tuple([])` i.e. empty list? will it just crash?

Reply:
if the tuple is empty list, then the bool clause will be false